use-timescale/security/ip-allow-list.md

```diff
@@ -0,0 +1,56 @@
+---
+title: IP allow list
+excerpt: Create a list of IP addresses that can access your services
+product: cloud
+keywords: [ip allow list, security]
+tags: [ip allow list]
+---
+
+# IP allow list
+
+You can restrict access to your $SERVICE_LONGs on public cloud to trusted IP addresses only. This prevents unauthorized connections without the need for a Virtual Private Cloud. Creating IP allow lists helps comply with security standards, for example, SOC 2 or HIPAA, that require IP filtering, especially in regulated industries like finance, healthcare, and government.
+
+<Highlight type="important">
+You can attach a $SERVICE_SHORT to either one VPC, or one IP allow list. 
+</Highlight>
+
+## Create and attach an IP allow list
+
+You create an IP allow list at the project level, then attach your $SERVICE_SHORT to it.
```

Comment:
Link to https://docs.timescale.com/use-timescale/latest/members/ on `project level` please.